WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsFormViewModel.swift

```diff
@@ -38,7 +38,7 @@ final class WooShippingCustomsFormViewModel: ObservableObject {
     @Published var itemsViewModels: [WooShippingCustomsItemViewModel] = []
 
     func onDismiss() {
-        // TODO: Add missing values if possible
+        // Package Id and Name not necessary to sync remotely
```

Comment:
I'm curious about this; won't we eventually need to include the package ID/name so we can support multiple shipments (where each shipment may have its own customs form)?

Reply:
Good point, I updated the comment here 0fafeb8b70